docker/pyramid/Dockerfile

```diff
@@ -56,12 +60,22 @@ COPY --chown=igvfd:igvfd src/igvfd/schema_formats.py src/igvfd/schema_formats.py
 
 COPY --chown=igvfd:igvfd src/igvfd/ontology.py src/igvfd/ontology.py
 
+COPY --chown=igvfd:igvfd src/igvfd/audit/* src/igvfd/audit/
```

Comment:
Can we put this as late a possible? Think audits change more than the stuff below. Maybe even after `RUN make-reference-db`?

Reply:
yep, pretty sure all of the audit doc stuff can go under that.

Reply:
moved everything down.